### DISCUSSION TITLE
Redirect from page to search

### DISCUSSION THREAD

**Discussion by Chimildic** [docsifyjs/docsify#1701] [Q&A]
Can I do redirect from page to search via `alias`? Example below doesn't work
```js
alias: {
    '/func?id=(.+)': '/?s=$1',
},
```

**Comment by trusktr** [comment]
I'm not sure. Also I think you meant `/?s=$1',`? Does that work?

**Reply by Chimildic** [reply]
Misspell. No, doesn't work.
